assets/dev/js/editor/components/settings/editor-preferences/manager.js

```diff
@@ -14,7 +14,10 @@ export default class extends BaseManager {
 			ui_theme: this.onUIThemeChanged,
 			panel_width: this.onPanelWidthChanged,
 			edit_buttons: this.onEditButtonsChanged,
+			show_hidden_elements: this.OnShowHiddenElementsChange,
 		};
+
+		elementor.on( 'globals:loaded', this.OnShowHiddenElementsChange );
```

Comment:
Don't run this here, add the appropriate classes where body classes for the Preview section are added.